sdk/component/internal/storer/local/sqlite/sqlite.go

```diff
@@ -50,31 +77,174 @@ func NewManager(dbPath string, dbName string) (*manager, error) {
 		return nil, fmt.Errorf("could not open sqlite db: %w", err)
 	}
 
-	return &manager{
+	mgr := &manager{
+		clock:  clockwork.NewRealClock(),
 		db:     db,
 		dbName: dbName,
-	}, nil
+	}
+
+	for _, opt := range opts {
+		if err := opt(mgr); err != nil {
+			return nil, fmt.Errorf("could not apply option: %w", err)
+		}
+	}
+
+	return mgr, nil
 }
 
-// TODO - implement me.
-func (m *manager) Read(ctx context.Context) ([]*ocsf.VulnerabilityFinding, error) {
-	return nil, nil
+// Validate. TODO - implement.
+func (m *manager) Validate(*ocsf.VulnerabilityFinding) error {
```

Comment:
As mentioned - let's work on this separately.

Reply:
can you make a ticket for this please to track it and add the ticket number in the comment?

Reply:
https://linear.app/smithy/issue/OCU-259/validate-component-input-and-output